fdb-relational-grpc/src/main/proto/grpc/relational/jdbc/v1/column.proto

```diff
@@ -49,7 +49,10 @@ message Struct {
 
 // Relational Array.
 message Array {
```

Comment:
I _think_ by making this a backward compatible change, and if the `elementType` is set (non-zero), the client will just get the type from the columns.
I _think_ this shouldn't be much work, and would allow us to keep the mixed-mode testing for returning arrays.

Reply:
Making the  changes allows `arrays.yamsql` to run, however `prepared.yamsql` still fails: The server-side code to deserialize (`FRL.java:254`) cannot handle an ARRAY type required by the tests.
We may patch this PR to the earlier branch if we want to allow these tests to run.